src/app/collapsible/collapsible.component.html

```diff
@@ -1,6 +1,7 @@
 <ul #collapsible
   class="collapsible"
   [class.popout]="popout"
+  [hidden]="items.length === 0"
 >
-  <ng-content select="mz-collapsible-item"></ng-content>
+  <ng-content></ng-content>
```

Comment:
Why the attribute `select` is removed?

Reply:
because it never been used anyway ... you already created a component `mz-collapsible-item`